src/integrations/prefect-ray/pyproject.toml

```diff
@@ -71,9 +70,6 @@ omit = ["tests/*", "prefect_ray/_version.py"]
 fail_under = 80
 show_missing = true
 
-[tool.uv.sources]
-prefect = { path = "../../../" }
-
```

Comment:
`ray==2.50.0` switched to using `uv` to set up execution environments, which doesn't work well with this source configuration. This change will cause the tests to run against the latest release `prefect` version instead of a dev version.